javascript-if/main.js

```diff
@@ -0,0 +1,78 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if ((person >= 16) && (person >= 21)) {
```

Comment:
`person` is meant to be an object, please re-read the instructions for this function.

Reply:
![Uploading Screen Shot 2022-07-09 at 2.34.17 PM.png…]()

Reply:
<img width="463" alt="Screen Shot 2022-07-09 at 2 34 17 PM" src="https://user-images.githubusercontent.com/97363006/178123497-f8a252c5-80b0-4611-9fca-6dc4e168a715.png">